### DISCUSSION TITLE
Select 2 not working

### DISCUSSION THREAD

**Discussion by eramitgupta** [livewire/livewire#6949] [1. Help]
When I am uploading files, like photos, and videos, select 2 is not working after that, how do I fix it?
please help me

https://drive.google.com/file/d/1o8aQkFPQQGhgqPkUy-KJeA3M2lGhijzE/view

**Comment by polycode-nz** [comment]
When the dom changes you'll loose your select2 instances unless you wire:ignore around them and handle it with events.

I use public properties on the component to store the values when changed.
